src/app-layout/visual-refresh-toolbar/toolbar/index.tsx

```diff
@@ -62,21 +95,22 @@ export function AppLayoutToolbarImplementation({
     toolbarState,
     setToolbarState,
     setToolbarHeight,
+    ...legacyProps
   } = appLayoutInternals;
   const {
-    ariaLabels,
-    activeDrawerId,
-    drawers,
-    drawersFocusRef,
-    onActiveDrawerChange,
-    hasNavigation,
-    navigationOpen,
-    navigationFocusRef,
-    onNavigationToggle,
-    hasSplitPanel,
-    splitPanelFocusRef,
-    splitPanelToggleProps,
-    onSplitPanelToggle,
+    ariaLabels = legacyProps.ariaLabels,
```

Comment:
I would suggest introducing a function to map legacy props to current props. This approach would provide a clearer representation of how each prop maps to the other. 
BTW, is this solution temporary? Do we have plans to deprecate the legacy API?

Reply:
Done.

> is this solution temporary

Yes, it is temporary, until all consumers will upgrade to the commit cf0f2b0755af1a28ac7c3c9476418a7ea807d0fd or newer